### DISCUSSION TITLE
Atomic Lava Lamp

### DISCUSSION THREAD

**Discussion by stretch911** [PlummersSoftwareLLC/NightDriverStrip#209] [Show and tell]
My copy cat lamp has been in prototype mode for testing and finish considerations. Just wanted to share a couple of those internal/external points of progress on this cool project.
![8375DAD2-29F6-4A2C-B987-6E1D5E85052E](https://user-images.githubusercontent.com/121883463/215781982-ba7981da-9f67-4938-978b-6f7b0e795ddb.jpeg)
![FA60E156-179B-4753-BB8C-0369B61C4DBF](https://user-images.githubusercontent.com/121883463/215781984-3e9a8ef2-ab98-44c1-9ce7-77c250f65ad8.jpeg)
![5A8F2CA5-86FC-490C-81BC-91EA2E089931](https://user-images.githubusercontent.com/121883463/215781987-2ae64fe9-238b-490e-ba5a-b03875150038.jpeg)
![9A6D3F79-1031-4757-BDF8-95F22E6CA6F5](https://user-images.githubusercontent.com/121883463/215781996-58a51ddd-2dac-4dc9-bdde-bb4cc63ca03b.jpeg)
![0CE1BCE3-2D23-46DA-86BB-7E38AA0AE8E2](https://user-images.githubusercontent.com/121883463/215781997-6fa9a9e0-3017-4625-a4be-6014076060b7.jpeg)
![EB8152EF-EB3B-4E0B-AEE1-1BC5CE56484D](https://user-images.githubusercontent.com/121883463/215782003-5f5c2b40-5549-4b54-b4ec-8c6280bb6816.jpeg)
![1A91EEB2-32F8-46EA-A583-F52FE2C108F9](https://user-images.githubusercontent.com/121883463/215782004-a1ad8112-def0-4831-b7f4-3ad1c2605750.jpeg)

**Comment by prschguy1** [comment]
Hello Stretch911, nice progress on the lamp. Looking great! I did a similar mast on mine as well. Looks more "atom like" separated from the base I think. While I was unable to find chrome tubing in the correct diameter, I did find some tubing that works well. Found this on Amazon : Stainless Steel 316L Welded Round Tubing, 1/2" OD, 0.46" ID, 0.020" Wall, 12" Length . Fits really well over the threaded rod. With a bit of wet sanding and metal polishing it gives it a chrome like finish and will never rust. As far as the sound reactive goes, still haven't got that working yet. Have tried line in, and all the electret boards I could find to no avail. Saw some info on Dave's Discord page for using the digital mics using I2s, so will try that next. I see that Dave and  rbergen have been working hard on the code and making lots of changes, so perhaps that will get fixed there. I really appreciate their dedication to the code . Interesting as it's ever evolving and almost organic in nature. Have learned a lot from working with it , watching what they are changing , and following the discussions. Very interesting.

**Reply by stretch911** [reply]
funny i was just thinking about that ugly threaded rod! thanks for the tips!
